Docs/developer_guide/build_instructions/linux.md

```diff
@@ -76,6 +76,41 @@ sudo apt update && sudo apt install git subversion build-essential cmake cmake-c
   qtbase5-private-dev libqt5x11extras5-dev libxt-dev
 ```
 
+### Ubuntu 23.04 (Lunar Lobster)
+
+:::{warning}
+If you get cmake version error, download build and install cmake 3.25.3:
+```console
+# Remove previous cmake versions
+sudo apt remove cmake
+sudo apt purge --auto-remove cmake
+
+sudo apt-get install build-essential
+wget https://cmake.org/files/v3.25/cmake-3.25.3.tar.gz
+tar xf cmake-3.25.3.tar.gz
+
+cd cmake-3.25.3
+./configure
+make
+# Then to install it to your system finally do 
+sudo make install 
+
+# Do this to refresh your console 
+hash -r
```

Comment:
```suggestion
```

Removing this section and including a note like we did for `Debian 12 Bookworm (Stable) and Bullseye 11 (OldStable)` should be sufficient.